### PR TITLE
fix: IOS Pushpins not set on location

### PR DESCRIPTION
**Pull request by LProulx86**
## Proposed Changes
Fix image placement for IOS Pushpin when icon aren't same size.

## What is the current behavior?
The most tiny pushpin isn't on location

## What is the new behavior?
All pushpin are on the location.

## Checklist

Please check if your PR fulfills the following requirements:

- [ ] Documentation has been added/updated
- [ ] Automated Unit / Integration tests for the changes have been added/updated
- [x] Contains **NO** breaking changes
- [ ] Updated the Changelog
- [ ] Associated with an issue

### CODE REVIEW

File: Samples/Samples/Samples.Shared/DynamicMap_FeaturesPage.xaml [hunk 1]
```diff
@@ -114,8 +114,10 @@
 									   AutolocateButtonVisibility="Collapsed"
 									   dynamicMap:MapControlBehavior.DisableRotation="True"
 									   dynamicMap:MapControlBehavior.PushpinImageSelector="{StaticResource PushpinEntityToMapPin}"
-									   dynamicMap:MapControlBehavior.SelectedIconWidth="32"
-									   dynamicMap:MapControlBehavior.SelectedIconHeight="50" />
+                                       dynamicMap:MapControlBehavior.IconWidth="36"
```
Comment (carlh98) [inline Samples/Samples/Samples.Shared/DynamicMap_FeaturesPage.xaml:117]: Please verify the format of the line

File: Samples/Samples/Samples.Shared/DynamicMap_FeaturesPage.xaml [hunk 2]
```diff
@@ -16,8 +16,8 @@
         <ResourceDictionary>
             <!--To make sure this works on iOS, these assets must exist not only with normal scale, but also 200 and 300 scale.-->
             <converters:FromPushpinEntityToStringConverter x:Key="PushpinEntityToMapPin"
-													   UnselectedValue="ms-appx:///Assets/Pushpin/inactive.png"
-													   SelectedValue="ms-appx:///Assets/Pushpin/active.png"/>
+                                                           UnselectedValue="ms-appx:///Assets/Pushpin/inactive.png"
+                                                           SelectedValue="ms-appx:///Assets/Pushpin/active.png"/>
```
Comment (carlh98) [inline Samples/Samples/Samples.Shared/DynamicMap_FeaturesPage.xaml:20]: space

File: src/Cartography/Cartography.DynamicMap/MapControlBehavior.cs [hunk 3]
```diff
@@ -20,106 +20,106 @@
 
 namespace Cartography.DynamicMap
 {
-	public static class MapControlBehavior
```
Comment (carlh98) [inline src/Cartography/Cartography.DynamicMap/MapControlBehavior.cs:23]: What changed in this file?

**Review by lesterbotello — APPROVED** [review]

File: src/Cartography/Cartography.DynamicMap/MapControlBehavior.cs [hunk 4]
```diff
@@ -131,96 +131,105 @@ private static void OnPusphinImageSelectorChanged(DependencyObject d, Dependency
...
+                    var selectedMargin = biggerHeight - selectedIconHeight;
+
+                    map.PinTemplate = view =>
+                    {
+                        var pushpin = view.Annotation as MapControlItem;
+
+                        // Moves the image up, so that the bottom of the pins points to the correct position.
+                        var yAxisPoint = (biggerHeight / 2);
+                        view.CenterOffset = new System.Drawing.PointF(0, -yAxisPoint);
+
```
Comment (carlh98) [inline src/Cartography/Cartography.DynamicMap/MapControlBehavior.cs:162]: space

**Pull request merged**